zerver/openapi/openapi.py

```diff
@@ -284,6 +284,17 @@ def generate_openapi_fixture(endpoint: str, method: str) -> list[str]:
                 subschema_status_code = status_code + "_" + str(subschema_index)
             else:
                 subschema_status_code = status_code
+
+            if get_schema(endpoint, method, status_code).get("x-no-response-body", False):
```

Comment:
I think rather than a special `x-no-response-body` value to check, just doing something different for `status_code in [301, 302]` seems right to me. The HTTP spec does not allow bodies in those responses, right?